packages/perseus/src/widgets/orderer/orderer.util.ts

```diff
@@ -0,0 +1,30 @@
+import type {
+    PerseusOrdererWidgetOptions,
+    PerseusRenderer,
+} from "@khanacademy/perseus-core";
+
+/**
+ * For details on the individual options, see the
+ * PerseusOrdererWidgetOptions type
+ */
+type OrdererPublicWidgetOptions = {
+    options: ReadonlyArray<PerseusRenderer>;
+    height: "normal" | "auto";
+    layout: "horizontal" | "vertical";
+};
+
+/**
+ * Given a PerseusOrdererWidgetOptions object, return a new object with only
+ * the public options that should be exposed to the client.
+ */
+function getOrdererPublicWidgetOptions(
+    options: PerseusOrdererWidgetOptions,
+): OrdererPublicWidgetOptions {
+    return {
+        options: options.options,
```

Comment:
I have a sense that we should "deep copy" these public variants. As this is, the `.options` array will be the same one between the original widget options and this public one we're returning. Likely never an issue, but making a copy feels safer. 

@benchristel do you have thoughts?

```suggestion
        options: [...options.options],
```

Reply:
Ben, didn't you mention that with Categorizer I think we update the values field based on user input? Would that be an instance of modifying these options? I think the main goal of these public widget options is just to render the clientside widgets, but if the object gets modified to track user input, copying might be needed.

Reply:
I can't see these functions ever being used directly with React. They'll be used in an editing environment and the publish pipeline most so the issue of mutation after copy doesn't matter here. 

So let's avoid copying.